Dockerfile

```diff
@@ -1,10 +1,5 @@
-# Below version includes rust 1.80.1
-ARG RUST_IMAGE_VERSION=@sha256:e8e40c50bfb54c0a76218f480cc69783b908430de87b59619c1dca847fdbd753
-# If you want to use latest version then uncomment next line
-# ARG RUST_IMAGE_VERSION=:slim-bookworm
-# Alternatively you can build docker with argument: --build-arg="RUST_IMAGE_VERSION=:slim-bookworm"
-
-FROM rust${RUST_IMAGE_VERSION} AS base-image
+# https://hub.docker.com/_/rust/tags
+FROM rust:1.81.0-slim-bookworm AS base-image
```

Comment:
The image sha under this tag is subject to change.
So might happen that our docker images will not be binary the same despite the same tag.

Reply:
> The image sha under this tag is subject to change.

Indeed - but the resultant image ( https://hub.docker.com/r/radixdlt/scrypto-builder/tags ) will be fixed / immutable, which is all I think that matters?

Reply:
The build of the resultant image will be not reproducible. If we can live with it then good to go!